test/evergreen/run_format_configs.sh

```diff
@@ -49,7 +49,7 @@ wait_for_process()
 
 				let "running--"
 
-				config_name=`egrep $process $tmp_file | awk -F ":" '{print $2}' | rev | awk -F "/" '{print $1}' | rev`
+				config_name=`egrep "\[${process}\]" $tmp_file | awk -F ":" '{print $2}' | rev | awk -F "/" '{print $1}' | rev`
```

Comment:
I think your original idea of matching full words with -w is probably better than making it []. Another option is "^${process}:" which could also work.

Reply:
@mickgraham-mongodb Fixed in 9d686a39e2d31ebaf50afd02086362ee12974560. Grepping for "^${process}:" would still fail because the `egrep` would return four entries for `egrep ^1791 format_list.txt`

1791:../../../test/format/failure_configs/CONFIG.WT-10017
17917:../../../test/format/failure_configs/CONFIG.WT-10075
17919:../../../test/format/failure_configs/CONFIG.WT-10258
17921:../../../test/format/failure_configs/CONFIG.WT-10384